### PR TITLE
Update requirements

### PR DESCRIPTION
**Pull request by mkgrey5921**
Feel free to make any changes

### CODE REVIEW

File: requirements.doc [hunk 1]
```diff
@@ -22,6 +22,8 @@ Functional:
              ii.	Rate of Decrease of Sugar Levels are increasing
 7.	The system should be able to inject insulin based off calculation
 8.	Alert/Sound user if there are any anomalies, and display a diagnostic/warning message
+          i. Create a ping system to locate the person using the (in case of emergency).
+           system when something goes wrong. Should have some sort of location services in case of any emergency.
```
Comment (HenryJGarcia) [inline requirements.doc]: Yeah I think that's a great idea, something calls 911 or emergency services when sugar levels are extremely low or extremely high.

File: requirements.doc [hunk 2]
```diff
@@ -22,12 +22,12 @@ Functional:
...
 7.	The system should be able to inject insulin based off calculation
 8.	Alert/Sound user if there are any anomalies, and display a diagnostic/warning message
-          i. Create a ping system to locate the person using the (in case of emergency).
-           system when something goes wrong. Should have some sort of location services in case of any emergency.
           a.	System Errors
           b.	Insulin resevoir running low
           c.	Blood sugar falling into the UNSAFE ZONE
-9.	System must show blood sugar levels on graph, and insulin dosage log
+          d.        Create a ping system to locate the person using the (in case of emergency) 
+                    system when something goes wrong. Should have some sort of location services in case of any emergency.
```
Comment (IsaiasCuevas) [inline requirements.doc]: @Nick-AI Thank you!

**Comment by Nick-AI** [comment]
Looks good to me, I suggest leaving it up until tonight just in case anybody wants to make changes and then we can merge!

**Comment by igree2737** [comment]
Looks really good. I tried to come up with a 10th functional requirement, but I think you covered all the important points.

**Comment by Nick-AI** [comment]
We have all of the requirements and it looks good to me, will merge the pull request!

**Pull request merged**